modules/swagger-jaxrs/src/main/java/com/wordnik/swagger/jaxrs/Reader.java

```diff
@@ -710,6 +735,31 @@ private static boolean isResourceClass(Class<?> cls) {
     return cls.getAnnotation(Api.class) != null;
   }
 
+  private List<Parameter> collectGlobalParameters(Class<?> cls) {
```

Comment:
This method looks like a candidate to put in a ReaderUtils class (or perhaps even ReflectionUtils) ?